mappings/net/minecraft/world/ServerWorldAccess.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5425 net/minecraft/world/ServerWorldAccess
+	COMMENT Represents access to a world on a logical Minecraft server.
+	METHOD method_30771 spawnEntityAndPassengers (Lnet/minecraft/class_1297;)V
+		COMMENT Spawns an entity and all it's passengers into the world.
+		ARG 1 entity
+	METHOD method_8410 getServerWorld ()Lnet/minecraft/class_3218;
```

Comment:
```suggestion
	METHOD method_8410 toServerWorld ()Lnet/minecraft/class_3218;
```
Like how java's `CompletionStage` has `toCompletableFuture`

Reply:
I can't check right now, but isn't this inconsistent with the other world access interfaces having "getWorld" instead?

Reply:
There is no other world access interface with this method. This is the sole one